layers/landcover/style.json

```diff
@@ -51,20 +39,13 @@
         "visibility": "visible"
       },
       "paint": {
-        "fill-color": "#6a4",
-        "fill-opacity": {
-          "base": 0.1,
+        "fill-color": {
```

Comment:
The fill color is very slightly darker now. But I think this is fine.

Reply:
Yeah, I was burning my eyes trying to figure out a tone close to the original enough at the different zoom levels and stopped at this point. It is a slight change, but if someone other than automation detects this I will send them my most sincere congratulations :sweat_smile: